packages/@aws-cdk/aws-codeguruprofiler/lib/profiling-group.ts

```diff
@@ -2,6 +2,20 @@ import { Grant, IGrantable } from '@aws-cdk/aws-iam';
 import { Construct, IResource, Lazy, Resource, Stack } from '@aws-cdk/core';
 import { CfnProfilingGroup } from './codeguruprofiler.generated';
 
+/**
+ * The compute platform of the profiling group.
+ */
+export enum ComputePlatform {
+  /**
+   * Use AWS_LAMBDA if your application runs on AWS Lambda.
+   */
+  AWS_LAMBDA = 'AWSLambda',
+  /**
+   * Use Default if your application runs on a compute platform that is not AWS Lambda, such an Amazon EC2 instance, an on-premises server, or a different platform.
+   */
+  DEFAULT = 'Default'
```

Comment:
Trailing comma here please!

```suggestion
  DEFAULT = 'Default',
```